### ISSUE TITLE
No theme or style system in the application

### ISSUE THREAD

**Issue by bsunderhus** [strvcom/dep-manager-web#24]
The application styles are a bit too chaotic, there's no theme defined.

Things like font, colors, sizes, and shapes (margin, etc,.) should be defined as a theme provider.

**Comment by dannytce** [comment]
@bsunderhus I would use ThemeProvider just in case if there will be more themes on a webapp (such ass dark/light mode). I am totally fine with plain javascript object. It's easier to follow and you don't need to worry that much about typing. Also DX is better, because of autocompletion etc.

**Comment by bsunderhus** [comment]
Sure thing the [ThemeProvider](https://www.styled-components.com/docs/api#themeprovider) is the best choice since this project already adopts _styled components_.

@dannytce it would be nice to have some style guiding based on sketches for the application before starting the refactoring of the components itself. Perhaps [zeplin](https://zeplin.io/)?

**Comment by dannytce** [comment]
But, there are no themes in app. Just forget about ThemeProvider and use plain javascript object ;)
